ilc/package.json

```diff
@@ -40,7 +40,8 @@
     "systemjs-css-extra": "^1.0.2",
     "tailorx": "^6.0.0",
     "url-join": "^4.0.1",
-    "uuid": "^3.4.0"
+    "uuid": "^3.4.0",
+    "wget-improved": "^3.2.1"
```

Comment:
probably it should be added to `devDependencies`, shouldn't it?

Reply:
I guess you are right
I moved it to dev
Thanks!